src/assets/icons/clue-dashboard/expand-group-view.svg

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <svg version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width="39px" height="30px" xmlns="http://www.w3.org/2000/svg">
   <g transform="matrix(1 0 0 1 -531 -563 )">
+    <title>Expand Group View</title>

```

Comment:
I'd rather not have the title in the SVG files; that just makes them less portable as assets and is redundant with the titles specified in the code.